components/server/ee/src/prebuilds/github-app.ts

```diff
@@ -341,8 +309,24 @@ export class GithubApp {
         return this.env.hostUrl.with({ pathname: '/button/open-in-gitpod.svg' }).toString();
     }
 
-    protected async findUserForInstallation(ctx: Context): Promise<User | undefined> {
-        const installation = await this.appInstallationDB.findInstallation("github", ctx.payload.installation.id);
+    protected async findInstallationOwner(installationId: number): Promise<{user: User, project?: Project} | undefined> {
+
+        // Project mode
+        //
+        const project = await this.projectDB.findProjectByInstallationId(String(installationId));
+        if (project) {
+            const owner = (await this.teamDB.findMembersByTeam(project.teamId)).filter(m => m.role === "owner")[0];
```

Comment:
> It could very well be a different user than whoever installed the app for a given project in that team.

This is a fair point. We haven't discussed the differentiation between installer and owner. I guess for now it's good enough, as there is no accounting for this.